src/TwitchLib.Communication/TwitchLib.Communication.csproj

```diff
@@ -22,6 +23,8 @@
         <FileVersion>2.0.0</FileVersion>
         <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
         <GenerateDocumentationFile>True</GenerateDocumentationFile>
+        <Features>strict</Features>
```

Comment:
```suggestion
```
Turns out we no longer need this and the warnings/errors will always be issued when newer language version features are used on an older target in a way which is not supported:
- https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/warning-waves
- https://learn.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.compilerfeaturerequiredattribute

It seems I had outdated information when introducing this, apologies.